tips/tests/test_rule_engine.py

```diff
@@ -113,10 +113,10 @@ def test_apply_rules_recursing(self):
     def test_stadspas(self):
         fixture = get_fixture()
         user_data = objectpath.Tree(fixture["data"])
-
         rules = [
             {"type": "ref", "ref_id": "1"} # ID 1 is the stadspas rule
         ]
+        print(user_data.execute("$.focus.*[@.soortProduct is 'Minimafonds' and @.typeBesluit is 'Toekenning']"))
```

Comment:
En wat is de output?

Reply:
zo kan je zien wat de regel doet, heeft Johan mij woensdag vertelt.

Reply:
Ja dat zie ik. Ik vroeg wat de output is. Dus wat wordt er geprint?

Reply:
<generator object Tree.execute.<locals>.exe.<locals>.exeSelector at 0x04CBB0F0>
Dit krijg ik terug

Reply:
Daar heb je weinig aan (-: Kun je uitvinden welke data daar achter schuil gaat?